common/api/components-react.api.md

```diff
@@ -3411,6 +3411,13 @@ export class UrlPropertyValueRenderer implements IPropertyValueRenderer {
 // @public
 export function useAsyncValue<T>(value: T | PromiseLike<T>): T | undefined;
 
+// @public
+export function useControlledTreeLayoutStorage<T extends Element>(): {
+    ref: React_2.RefObject<T>;
```

Comment:
Could we change the `ref` type of these hooks to `Ref<T>`?

Reply:
Changed to `Ref<T>`